server/mongo_data_access.go

```diff
@@ -14,16 +14,67 @@ import (
 	"gopkg.in/mgo.v2/bson"
 )
 
+// MongoConnection manages the direct connection to the Mongo database
+type MongoConnection struct {
+	session *mgo.Session
+	dbname  string
+}
+
+// Connect establishes a session with the mongo database
+func (dc *MongoConnection) Connect(host string) (err error) {
```

Comment:
Yeah, once upon a time I gave it the more ambiguous name "DatabaseConnection". Would you like me to change this?